libs/designsystem/form-field/src/input-counter/input-counter.component.spec.ts

```diff
@@ -29,7 +29,7 @@ describe('InputCounterComponent', () => {
     });
 
     it('should render counter correctly', () => {
-      expect(spectator.element.textContent).toEqual('');
+      expect(spectator.element.textContent.replace(/(\r\n|\n|\r)/gm, '')).toEqual('');
```

Comment:
Be aware of this fix and also why it was needed. This is done to prevent the test from failing due to the added linebreaks in the template.

Basically, prettier will format the HTML which actually adds newlines in the template. The tests then compares the content from the template with a test value which no longer matches as the value retrieved from the component now includes the newlines added by prettier :-)

Reply:
Could spectator.element.textContent.trim() be an alternative here, for better readability?